### ISSUE TITLE
Branch status should automatically updated in status bar.

### ISSUE THREAD

**Issue by dli-odoo** [microsoft/vscode#75264]
> When using the terminal for checkout branch in remote ssh development, checking out branch does not update bottom left side branch name. I think it should automatically be synced and updated in the status bar also if I do it by the terminal on visual studio code.

> As I noticed, I check out to master branch from another branch, but it was showing another branch in the status bar, not another branch. But that should be synced and automatically updated in remote development.

> IMHO, it would be more user-friendly.

**Comment by joaomoreno** [comment]
This https://github.com/microsoft/vscode/commit/13d0dad699bfc4e4529ece16dd577be98945fce1 should've fixed it. Please try it again tomorrow on Insiders and let me know if it works for you.

**Comment by dli-odoo** [comment]
@joaomoreno let me give you some more information here.

I just installed the latest updates on Insiders and checked. but that the feature is not there.

Let me give you some more specification:

As we connect to the remote server for development using ssh extension. In that case, the terminal we use on insiders also shows the directory and path of the ssh server.

So when I try to switch the branch from the terminal in git, like ex: I am in master branch. and I want to move into another branch say issue-branch-me. But as I am doing it from the terminal, and not from the status bar.

So here the case happens that I am in the issue-branch-me branch but in the status bar it shows the master branch and not updated to the branch I moved in.

IMHO, that should be tracked and updated in the status bar also. because for me it becomes quite problematic.

Hope this helps to understand more. Thank you. :)
